nixos/modules/virtualisation/containers.nix

```diff
@@ -518,6 +518,7 @@ in
         KillSignal = "WINCH";
 
         DevicePolicy = "closed";
+        DeviceAllow = "/dev/net/tun rw";
```

Comment:
Maybe adding a list of allowed devices (with a default value?), would be helpful, so people can decide them self, what to allow.